src/File.cpp

```diff
@@ -110,7 +132,40 @@ std::vector<std::string> TextFile::readlines(size_t n) {
     if (this->fail()) {
         throw file_error("could not read a line in {}", this->path());
     }
+
     // Re-enable exceptions checking
     this->exceptions(state);
+
     return lines;
 }
+
+void TextFile::skipline() {
+    // Disable exceptions checking, and manually check bellow
+    auto state = this->exceptions();
+    this->exceptions(std::fstream::goodbit);
+
+    skip_line();
```

Comment:
If the intent is to make `skip_line` private, would `skipline_` be a better name? This matches the convention used by private class members.